server/GraknServer.java

```diff
@@ -127,6 +130,13 @@ private Path dataDir() {
         return command.dataDir();
     }
 
+    private void configureAndVerifyJavaVersion() {
+        int javaMajorVersion = Java.getMajorVersion();
+        if (javaMajorVersion < 11) {
```

Comment:
I think there's no need for the variable:

```suggestion
        if (Java.getMajorVersion() < 11) {
```

Reply:
@lolski it's extracted into a variable to be later passed into an error message:

```
            throw GraknException.of(INCOMPATIBLE_JAVA_RUNTIME, javaMajorVersion);
```